src/rttest.cpp

```diff
@@ -728,7 +781,7 @@ std::string rttest_results_to_string(struct rttest_results * results, char * nam
   sstring << "    - Standard deviation: " << results->latency_stddev << std::endl;
   sstring << std::endl;
 
-  return sstring.str();
+  return sstring.str().c_str();
```

Comment:
I've added a test case that shows that this function is fine (I think because it returns a `const char *`). Let me know what you think of this:

https://github.com/ros2/rttest/pull/8/files#diff-7a77cf3df4d541d275312750639d96c1R162